test/unit/ansible_role_test.rb

```diff
@@ -2,6 +2,10 @@
 
 # Tests for the behavior of Ansible Role, currently only validations
 class AnsibleRoleTest < ActiveSupport::TestCase
+  setup do
```

Comment:
What is this needed for? It's creating a role on every should of this test with no purpose